src/ripple/beast/container/detail/aged_ordered_container.h

```diff
@@ -163,9 +163,17 @@ class aged_ordered_container
     // VFALCO TODO This should only be enabled for maps.
     class pair_value_compare
         : public boost::beast::detail::empty_base_optimization <Compare>
+#ifdef OS_MACOSX
```

Comment:
This is really more about libc++ than macOS.  Perhaps `#ifdef _LIBCPP_VERSION` would be more appropriate.  Not a big deal though.